dockers/tpu-tests/Dockerfile

```diff
@@ -1,37 +0,0 @@
-# Copyright The PyTorch Lightning team.
```

Comment:
I think we need them or how is this related to dropping 1.9?

Reply:
let's have it still as a separate PR :)

and I think they are used in our TPU testing: https://github.com/Lightning-AI/lightning/blob/7decc50a879c795821927709790bb515916ef99b/dockers/tpu-tests/Dockerfile#L18

Reply:
Done in https://github.com/Lightning-AI/lightning/pull/15611